deltachat-ios/Controller/MediaQualityController.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+import DcCore
+class MediaQualityController: UITableViewController {
+
+    var dcContext: DcContext
+
+    var options: [Int]
+
+    var staticCells: [UITableViewCell] {
+        return options.map({
+            let cell = UITableViewCell(style: .default, reuseIdentifier: nil)
+            cell.textLabel?.text = MediaQualityController.getValString(val: $0)
+            cell.selectionStyle = .none
+            return cell
+        })
+    }
```

Comment:
Better use `lazy + closure`-syntax here, so the array is built only once. This way (computed variable) it is very inefficent (O(n<sup>2</sup>))  when populating the cells. It will basically create a fresh array each time you access staticCells for instance `staticCell[n]`.

Reply:
fixed, thanks for the explanation